src/libs/actions/PersonalDetails.js

```diff
@@ -278,7 +278,7 @@ function setPersonalDetails(details, shouldGrowl) {
         .then((response) => {
             if (response.jsonCode === 200) {
                 if (details.timezone) {
-                    NameValuePair.set(CONST.NVP.TIMEZONE, details.timezone);
+                    User.setPreferredTimezone(details.timezone);
```

Comment:
This seems like "chaining" and not something we want. Is `PersonalDetails_Update` being handled elsewhere? I would expect to see:

- `API.write('UpdatePersonalDetails')`
- And on the server the timezone would be set and not done in a separate API call

Reply:
Removed this change